frontend/cypress/integration/common/app_details.ts

```diff
@@ -83,11 +83,11 @@ Then('no cluster badge for the {string} should be visible', (type: string) => {
   if (type === 'Istio config') {
     cy.get('#pfbadge-C').should('not.exist');
   } else if (type === 'graph side panel') {
-    cy.get('#graph-side-panel').within($div => {
+    cy.get('#graph-side-panel').within(() => {
       cy.get('#pfbadge-C').should('not.exist');
     });
   } else {
-    cy.get(`#${type[0].toUpperCase()}${type.slice(1)}DescriptionCard`).within($article => {
+    cy.get(`#${type[0].toUpperCase()}${type.slice(1)}DescriptionCard`).within(() => {
```

Comment:
I find `.within($article => { ... })` valid here

Reply:
Yes, it is valid, but the $article input variable is not used within the arrow function. VS code linting tool encourage to remove unused variables and function parameters.